### PR TITLE
Rh tabbed interface

### PR DESCRIPTION
**Pull request by rezahusain**

### CODE REVIEW

File: config/routes.rb
Comment (jrzricardo) [inline config/routes.rb:1]: new routes lookin nice.

File: app/views/users/show.html.erb
Comment (jrzricardo) [inline app/views/users/show.html.erb:1]: including partial in your render is optional, but I respect your choice in including it :>

File: app/views/users/liked.html.erb
Comment (jrzricardo) [inline app/views/users/liked.html.erb:1]: Ricardo liked this.

File: app/views/users/following.html.erb
Comment (jrzricardo) [inline app/views/users/following.html.erb:1]: we got bars.

File: app/views/users/followers.html.erb
Comment (jrzricardo) [inline app/views/users/followers.html.erb:1]: bars we have.

File: app/views/users/feed.html.erb
Comment (jrzricardo) [inline app/views/users/feed.html.erb:1]: barred up.

File: app/views/shared/_user_navbar.html.erb
Comment (jrzricardo) [inline app/views/shared/_user_navbar.html.erb:1]: nice job associating the proper paths

File: app/views/photos/_photo.html.erb
Comment (jrzricardo) [inline app/views/photos/_photo.html.erb:1]: the div you have that includes  "card link" and "another card" should probably be removed. but everything else looks good.

File: app/views/comments/_form.html.erb
Comment (jrzricardo) [inline app/views/comments/_form.html.erb:1]: lookin clean.

File: app/controllers/comments_controller.rb
Comment (jrzricardo) [inline app/controllers/comments_controller.rb:1]: we love a come back.

**Review by jrzricardo — APPROVED** [review]
good looking code my dude

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job creating a partial for the navbar! You can now merge these changes back to main.

File: app/views/shared/_user_navbar.html.erb
Comment (FadiBaker92) [inline app/views/shared/_user_navbar.html.erb:1]: great job on creating the navbar partial

**Comment by FadiBaker92** [comment]
Looks great Reza, good job!

**Pull request merged**